app/models/user.rb

```diff
@@ -958,6 +958,10 @@ def silenced_at
     silenced_record.try(:created_at) if silenced?
   end
 
+  def silenced_forever?
+    silenced_till > DateTime.now + 500.years
```

Comment:
Considering the current longevity of website and humans, 100 years should be enough?

```suggestion
    silenced_till > 100.years.from_now
```

Reply:
Hmm, when suspending forever we're suspending even for 1000 years :) But yes 100 years here is better.